controllers/controllers.articles.js

```diff
@@ -54,4 +54,12 @@ const patchArticleVotes = (request, response, next) => {
     .catch(next);
 };
 
-module.exports = {getArticlePath, getArticleById, getCommentsByArticleId, postComment, patchArticleVotes}
\ No newline at end of file
+const getUsers = (request, response, next) => {
+  selectUsers().then((users)=> {
+        response.send({users})
+    })
+    .catch(next)
+  }
+
+
+module.exports = {getArticlePath, getArticleById, getCommentsByArticleId, postComment, patchArticleVotes, getUsers}
```

Comment:
Im not sure why there are two of these. There another in controllers.users.js

The other one is better - it sends the status

Reply:
Yeah I was getting an error and was trying to sort it out and then realised it was coming from the test and tried to see if there was something wrong with the new controller I had set up. Will remove this function.